src/main/scala/nodes/nlp/POSTagger.scala

```diff
@@ -0,0 +1,36 @@
+package nodes.nlp
+
+import epic.sequences.{CRF, TaggedSequence}
+import epic.trees.AnnotatedLabel
+import workflow.Transformer
+import org.apache.spark.rdd.RDD
+
+/**
+  * Transformer that uses Epic to part-of-speech tag a sequence of words.
+  * It is recommended to use a Tokenizer before applying the POS Tagger,
+  * assuming we get a list of strings as input.
+  *
+  * Note: Epic model is kind of heavy (~300MB) so it may take some time to load
+  * TODO: Test the model on a Spark cluster w/ and w/o broadcast variable for benchmark
+  *
+  * Here's an example:
+  * {{{
+  *   val model = epic.models.PosTagSelector.loadTagger("en").get
+  *   val POSTagger = POSTagger(model).apply(data)
+  * }}}
+  *
+  * @param model The POS Tagger model loaded from the Epic library
+  */
+case class POSTagger(model: CRF[AnnotatedLabel, String])
```

Comment:
Also, now that we're broadcasting this model, I think it makes sense to mark this as `@transient`, because we explicitly handle sending the model to the workers and we don't want the model getting picked up by Spark's closure capture.